api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -751,4 +758,39 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
     resp.setItems(userRoles);
     return ResponseEntity.ok(resp);
   }
+
+  @Override
+  public ResponseEntity<WorkspaceResponseListResponse> getPublishedWorkspaces() {
+    WorkspaceResponseListResponse response = new WorkspaceResponseListResponse();
+
+    return ResponseEntity.ok(response);
+  }
+
+  @Override
+  @AuthorityRequired({Authority.WORKSPACE_ADMIN})
```

Comment:
Could we call this authority `FEATURED_WORKSPACE_ADMIN`? I imagine there are other things that will be separate workspace actions. (Like reviewing research purposes)

This is more about publishing than it is administering workspaces.